vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -51,14 +52,18 @@ type VolumeDriver struct {
 	refCounts  *refcount.RefCountsMap
 }
 
-var mountRoot string
+var (
+	mountRoot         string
+	fullVolumeNameMap map[string]string // save full volume names to avoid trip to vmdkops service
```

Comment:
for all maps , please comment what do they map (key) and to what (value). E.g. volumeMoundID->fullVolumeName